_deploy_event/configmap.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: "{{{APPLICATION}}}-{{{COMPONENT}}}"
```

Comment:
Any reason to not hardcode this values?

Reply:
No particular reason. Changed it.

Reply:
Hm, maybe we need APPLICATION var delivery.yaml for CDP to figure out application...

Reply:
Good point, I didn't actually validate what the event looks like :facepalm: Let me double check it.